packages/web3-common/src/errors.ts

```diff
@@ -482,10 +482,3 @@ export class PBKDF2IterationsError extends Web3Error {
 		super('c > 1000, pbkdf2 is less secure with less iterations');
 	}
 }
-
```

Comment:
For Removing circular dependency https://github.com/ChainSafe/web3.js/pull/4742/commits/6b752ba74246d270a2b6e177b234e4ae69d9e5b2